Bottom: f150652ca82a5866f82c590caaa5312f4a3a62bf
Top:    7245b1f142c9e2c56230ccee03ebe9e8abad0045
Author: Ian Kent <ikent@redhat.com>
Date:   2013-04-03 15:42:29 +0800

Refresh of debug.patch

---

diff --git a/daemon/automount.c b/daemon/automount.c
index 4c651cf..1da00df 100644
--- a/daemon/automount.c
+++ b/daemon/automount.c
@@ -1345,10 +1345,12 @@ static void *statemachine(void *arg)
 			if (list_empty(&master_list->completed)) {
 				if (list_empty(&master_list->mounts)) {
 					master_mutex_unlock();
+					debug(LOGOPT_ANY, "list empty completed and mounts");
 					return NULL;
 				}
 			} else {
 				if (master_done(master_list)) {
+					debug(LOGOPT_ANY, "master_done returns true");
 					master_mutex_unlock();
 					return NULL;
 				}
